app/src/main/java/com/framgia/soundclound/data/model/Track.java

```diff
@@ -0,0 +1,256 @@
+    @Expose
+    private int mDownloadCount;
+    @SerializedName("download_url")
+    @Expose
+    private Object mDownloadUrl;
+    @SerializedName("duration")
+    @Expose
+    private int mDuration;
+    @SerializedName("full_duration")
+    @Expose
+    private int mFullDuration;
+    @SerializedName("genre")
+    @Expose
+    private String mGenre;
+    @SerializedName("has_downloads_left")
+    @Expose
+    private boolean mHasDownloadsLeft;
+    @SerializedName("id")
+    @Expose
+    private int mId;
+    @SerializedName("kind")
+    @Expose
+    private String mKind;
+    @SerializedName("likes_count")
+    @Expose
+    private int mLikesCount;
+    @SerializedName("permalink_url")
+    @Expose
+    private String mPermalinkUrl;
+    @SerializedName("playback_count")
+    @Expose
+    private int mPlaybackCount;
+    @SerializedName("publisher_metadata")
+    @Expose
+    private PublisherMetadata mPublisherMetadata;
+    @SerializedName("title")
+    @Expose
+    private String mTitle;
+    @SerializedName("uri")
+    @Expose
+    private String mUri;
+    @SerializedName("user_id")
+    @Expose
+    private int mUserId;
+    @SerializedName("display_date")
+    @Expose
+    private String mDisplayDate;
+    @SerializedName("user")
+    @Expose
+    private User mUser;
+
+    public Track(String title) {
+        mTitle = title;
+    }
+
+    public Track() {}
+
+    public String getArtworkUrl() {
+        return mArtworkUrl;
+    }
+
+    public void setArtworkUrl(String artworkUrl) {
+        mArtworkUrl = artworkUrl;
+    }
+
+    public int getCommentCount() {
+        return mCommentCount;
+    }
+
+    public void setCommentCount(int commentCount) {
+        mCommentCount = commentCount;
+    }
+
+    public String getDescription() {
+        return mDescription;
+    }
+
+    public void setDescription(String description) {
+        mDescription = description;
+    }
+
+    public boolean getDownloadable() {
+        return mDownloadable;
+    }
+
+    public void setDownloadable(boolean downloadable) {
+        mDownloadable = downloadable;
+    }
+
+    public int getDownloadCount() {
+        return mDownloadCount;
+    }
+
+    public void setDownloadCount(int downloadCount) {
+        mDownloadCount = downloadCount;
+    }
+
+    public Object getDownloadUrl() {
+        return mDownloadUrl;
+    }
+
+    public void setDownloadUrl(Object downloadUrl) {
+        mDownloadUrl = downloadUrl;
+    }
+
+    public int getDuration() {
+        return mDuration;
+    }
+
+    public void setDuration(int duration) {
+        mDuration = duration;
+    }
+
+    public int getFullDuration() {
+        return mFullDuration;
+    }
+
+    public void setFullDuration(int fullDuration) {
+        mFullDuration = fullDuration;
+    }
+
+    public String getGenre() {
+        return mGenre;
+    }
+
+    public void setGenre(String genre) {
+        mGenre = genre;
+    }
+
+    public boolean getHasDownloadsLeft() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#BooleanGetMethodName